src/screens/OrganizationDashboard/OrganizationDashboard.tsx

```diff
@@ -94,24 +97,20 @@ function OrganizationDashboard(): JSX.Element {
             <div className={styles.sidebarsticky}>
               <h6 className={styles.titlename}>{t('about')}</h6>
               <p className={styles.description}>
-                {data.organizations[0].description}
+                {data?.organizations[0].description}
               </p>
               <p className={styles.toporgloc}>
-                {t('location')} : {data.organizations[0].location}
+                {t('location')} : {data?.organizations[0].location}
```

Comment:
This condition suggests that there instances where data is `null` or `undefined`. That said, what will you display if that happens.

Reply:
Hi @Rumone, actually the `data` can be `undefined` only when the `id` is invalid which case is already being handled by the `error` field.

The problem here is asynchronous state changes - 

- We fetch the Organization, data is initially `undefined` while `loading` becomes true
- Once the data is loaded, `loading` becomes false and we show the `data` which now has a value
- And in case something goes wrong, `error` is set to the error, and `data` is still `undefined`
- We are handling the error case before accessing `data`

Thus, theoretically `data` cannot be `undefined` unless there's an error from the backend which we have already handled using `error`  before accessing `data`

The problem is the value of `data` is not immediately set after `loading` becomes false

The conditional check is only to wait for the value to be updated by Apollo Client

I don't think we need to display it in this condition

Reply:
@Rumone can we resolve this?